### ISSUE TITLE
Description of 'employee' Object setup seems inaccurate. (Grammar?)

### ISSUE THREAD

**Issue by brigj1** [learn-co-curriculum/phase-0-intro-to-js-2-objects-lab#7]
### Canvas Link

https://learning.flatironschool.com/courses/5506/assignments/170669?module_item_id=371063

### Concern

Grammar here seems inaccurate:
To start, define a employee variable and assign it to an Object containing name and streetAddress keys; 

This sounds like we are assigning a variable to an Object. But don't we assign stuff to variables? That is, it seems like we should be assigning an Object to the variable 'employee', and the Object should have keys of 'name' and 'streetAddress'.

### Suggested Changes

Maybe rephrase above to:

To start, define a employee variable and assign an Object to it that has keys of name and streetAddress;

**Comment by professor-ben** [comment]
Hi Brigham,

I agree with your assessment here. I just merged a fix into master.

Thanks,
Ben
